sdks/java/io/kafka/build.gradle

```diff
@@ -71,7 +71,10 @@ dependencies {
   implementation library.java.joda_time
   implementation library.java.jackson_annotations
   implementation library.java.jackson_databind
-  implementation group: 'com.google.cloud.hosted.kafka', name: 'managed-kafka-auth-login-handler', version: '1.0.5'
+  implementation (group: 'com.google.cloud.hosted.kafka', name: 'managed-kafka-auth-login-handler', version: '1.0.5') {
```

Comment:
@fozzie15 could you review this? And could you follow up and add a test to help us avoid breakages like this in the future (seems like a hard to predict issue, but we could have a test where a user brings their own kafka library, or we could just add this path to https://github.com/apache/beam/blob/master/.github/workflows/beam_PreCommit_SQL.yml)